jvb/rootfs/defaults/sip-communicator.properties

```diff
@@ -21,3 +21,11 @@ org.jitsi.videobridge.xmpp.user.shard.DISABLE_CERTIFICATE_VERIFICATION=true
 org.jitsi.videobridge.ENABLE_STATISTICS=true
 org.jitsi.videobridge.STATISTICS_TRANSPORT=muc
 org.jitsi.videobridge.STATISTICS_INTERVAL=5000
+
+{{ $ENABLE_OCTO := .Env.ENABLE_OCTO | default "0" | toBool }}
+{{ if $ENABLE_OCTO }}
+org.jitsi.videobridge.octo.BIND_ADDRESS={{ .Env.JVB_NAT_HARVESTER_LOCAL_ADDRESS }}
+org.jitsi.videobridge.octo.PUBLIC_ADDRESS={{ .Env.JVB_NAT_HARVESTER_PUBLIC_ADDRESS }}
+org.jitsi.videobridge.octo.BIND_PORT=4096
+org.jitsi.videobridge.REGION=Region01
```

Comment:
make `org.jitsi.videobridge.octo.BIND_PORT` and `org.jitsi.videobridge.REGION` configurable too using env variables.

org.jitsi.videobridge.octo.BIND_PORT= {{ .Env.OCTO_BIND_PORT }}
org.jitsi.videobridge.REGION= {{ .Env.OCTO_REGION }}

Reply:
Done